qiskit/test/decorators.py

```diff
@@ -90,7 +90,9 @@ def _get_credentials():
                                                        discover_credentials)
     except ImportError:
         raise unittest.SkipTest('qiskit-ibmq-provider could not be found, '
-                                'and is required for executing online tests.')
+                                'and is required for executing online tests. '
+                                'Install iskit-ibmq-provider or check your '
```

Comment:
Could you update this message to say how to install the IBMQ provider, like how the other messages are? `"pip install qiskit-ibmq-provider"`